pkgs/stdenv/generic/setup.sh

```diff
@@ -355,7 +388,9 @@ _addToNativeEnv() {
     runHook envHook "$pkg"
 }
 
-for i in "${nativePkgs[@]}"; do
+set +u # Old bash thinks empty array == undefined array
```

Comment:
Ah I forgot to remove this `set +u`now that I do `${nativePkgs[@]:-"${nativePkgs[@]}":-}`. Not sure this is worth restarting @globin's job for, though.